.github/workflows/update-mds.yml

```
@@ -0,0 +1,43 @@
+name: Update README.md
+
+on:
+  pull_request:
+
+    branches:
+      - main
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

https://github.com/actions/checkout?tab=readme-ov-file#Checkout-pull-request-HEAD-commit-instead-of-merge-commit

```suggestion
        with:
          fetch-depth: 0
          ref: ${{ github.event.pull_request.head.sha }}
```